src/api/api.ts

```diff
@@ -0,0 +1,19 @@
+import { Todo, User } from '../react-app-env';
+
+const API_URL = 'https://mate.academy/students-api';
+
+const request = (url: string) => {
+  return fetch(`${API_URL}${url}`)
```

Comment:
unless a library or legacy codebase forces you to use then/catch, the better choice for readability and maintainability is async/await